libs/camera-streamer.sh

```diff
@@ -36,11 +36,21 @@ function run_ayucamstream() {
     rtsp=$(get_param "cam ${cam_sec}" enable_rtsp)
     rtsp_pt=$(get_param "cam ${cam_sec}" rtsp_port)
     cstm="$(get_param "cam ${cam_sec}" custom_flags 2> /dev/null)"
+    noprx="$(get_param "crowsnest" no_proxy 2> /dev/null)"
     ## construct start parameter
     # set http port
     #
     start_param=( --http-port="${pt}" )
 
+    if [[ -n "${noprx}" ]] && [[ "${noprx}" = "true" ]]; then
+        # See https://github.com/mainsail-crew/crowsnest/pull/221#issuecomment-1863555700 for why
+        # we cannot assume the binary has support for `--http-listen`.
+        if $ust_bin --help | grep -q "http-listen"; then
+            start_param+=( --http-listen "0.0.0.0" )
```

Comment:
Did you actually test this? At least it doesn't work for me like this. You are missing a `=` here.
`--http-listen="0.0.0.0"` would be correct. I would even write `--http-listen=0.0.0.0`

Reply:
You're right, locally I had just added `--http-listen="0.0.0.0"`

![image](https://github.com/mainsail-crew/crowsnest/assets/2403652/2a7fe0b1-b215-4dd1-976a-90a17eaee75f)